pyomo/solvers/plugins/converter/glpsol.py

```diff
@@ -104,7 +104,7 @@ def apply(self, *args, **kwargs):
                 OUTPUT.write("end;\n")
             OUTPUT.close()
             cmd.append(modfile)
-        pyutilib.subprocess.run(cmd)
+        subprocess.run(cmd)
```

Comment:
Should this have `stdout=subprocess.DEVNULL, stderr=DEVNULL`, because the default behavior of the `pyutilib.subprocess.run` was to suppress the command output?